src/utils.jl

```diff
@@ -10,10 +12,12 @@ function gitcmd(path::AbstractString, gitconfig::Dict)
 end
 
 """
-    registration_branch(pkg::Pkg.Types.Project) -> String
+    registration_branch(pkg::Pkg.Types.Project; url::String) -> String
```

Comment:
Why is it a keyword argument? It looks very mandatory to me

Reply:
Yeah it is required. I just thought it would be nice to have a named parameter. I'm fine either way (positional argument versus required keyword argument).